.github/workflows/publish.yaml

```diff
@@ -33,12 +32,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          # git config http.sslVerify false
-          # repo="https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
           git config --local user.email "ersilia-bot@users.noreply.github.com"
           git config --local user.name "ersilia-bot"
           git tag -a "v${{ steps.version.outputs.VERSION }}" -m "v${{ steps.version.outputs.VERSION }}"
-          # git push "${repo}" origin "v${{ steps.version.outputs.VERSION }}"
+          git push origin "v${{ steps.version.outputs.VERSION }}"
```

Comment:
I think we can remove this line entirely. I added it when I was trying to push tags through a script but that happens implicitly when we set `tags` to true in the commit and push action.

Reply:
There is no explicit git push for tags again. It is being handled implicitly when `tags: true ` is set in the commit and push action.